README.md

```diff
@@ -36,41 +38,134 @@ $bundles = [
 
 ### Step 3: Configure plugin
 ```yaml
-# config/packages/_sylius.yaml
+# config/packages/setono_product_callout.yaml
 
 imports:
     - { resource: "@SetonoSyliusCalloutPlugin/Resources/config/app/config.yaml" }
+
+setono_sylius_callout:
+    manual_triggering: false
+    # Enable manual triggering if your store have too much products
+    # That way you can trigger callouts assign process manually when
+    # finish adding all rules
+    # manual_triggering: true
 ```
 
 ### Step 4: Import routing
 
 ```yaml
-# config/routes/routes.yaml
+# config/routes/setono_product_callout.yaml
 
 setono_product_callout:
     resource: "@SetonoSyliusCalloutPlugin/Resources/config/routing.yaml"
 ```
 
-### Step 5: Customize models and repositories
-5. Customize your product model. Read more about Sylius models customization [here](https://docs.sylius.com/en/latest/customization/model.html).
-- add a `Setono\SyliusCalloutPlugin\Model\CalloutsAwareTrait` trait to your `App\Entity\Product` class (check our [this path](tests/Application/src) for a reference),
-- add callouts relation to your `Product.orm.xml` like [here](tests/Application/src/Resources/config/doctrine),
-- if you haven't done so already, configure the `sylius_product` resource to point to your `App\Entity\Product` like we 
-did in an example [here](tests/Application/src/Resources/config/resources.yml).
+### Step 5: Customize models
+
+Read more about Sylius models customization [here](https://docs.sylius.com/en/latest/customization/model.html).
+
+#### Customize your Product model
+
+Add a `Setono\SyliusCalloutPlugin\Model\CalloutsAwareTrait` trait to your `App\Entity\Product` class.
+
+- If you use `annotations` mapping:
+
+    ```php
+    <?php 
+    // src/Entity/Product.php
+    
+    namespace App\Entity;
+
+    use Setono\SyliusCalloutPlugin\Model\CalloutsAwareTrait as SetonoSyliusCalloutPluginCalloutsAwareTrait;
+    use Setono\SyliusCalloutPlugin\Model\ProductInterface as SetonoSyliusCalloutPluginCalloutsProductInterface;
+    use Sylius\Component\Core\Model\Product as BaseProduct;
+    use Doctrine\ORM\Mapping as ORM;
+    
+    /**
+     * @ORM\Entity
+     * @ORM\Table(name="sylius_product")
+     */
+    class Product extends BaseProduct implements SetonoSyliusCalloutPluginCalloutsProductInterface
+    {
+        use SetonoSyliusCalloutPluginCalloutsAwareTrait {
+            SetonoSyliusCalloutPluginCalloutsAwareTrait::__construct as private __calloutsTraitConstruct;
+        }
+      
+        public function __construct()
+        {
+            $this->__calloutsTraitConstruct();
+            parent::__construct();
+        }
+    }
+    ```
+    
+- If you use `xml` mapping:
```

Comment:
In my opinion we should not clutter the README with too much info. If you use XML mapping my guess is you can convert the annotations to XML yourself. What do you think?

Reply:
For sure can, but it much easier to copy-paste. I believe we should keep `annotations` for devs as it default and `xml` at least for us for easier/faster integration to our projects.